### PR TITLE
Setup instructor config, service

### PR DESCRIPTION
**Pull request by bharaninb**

### CODE REVIEW

File: packages/instructors/.eslintrc [hunk 1]
```diff
@@ -7,7 +7,8 @@
     "react/prefer-stateless-function": 0,
     "import/no-extraneous-dependencies": ["error", {
       "devDependencies": true
-    }]
+    }],
+    "no-underscore-dangle": [2, { "allow": ["_id"] }]
```
Comment (ackinc) [inline packages/instructors/.eslintrc]: Better to replace the '2' with the actual string

File: packages/instructors/src/services/cookie.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export const getCookie = (key) => {
```
Comment (ackinc) [inline packages/instructors/src/services/cookie.js]: We should put these services in a 'frontend-services' folder in the project root and import them into students/instructors

File: packages/instructors/.eslintrc (hunk 1 above)
Comment (bharaninb) [reply packages/instructors/.eslintrc]: Updated

File: packages/instructors/src/services/cookie.js (hunk 2 above)
Comment (bharaninb) [reply packages/instructors/src/services/cookie.js]: Creating separate PR for this change.

**Pull request merged**